content/en/logs/guide/aws-eks-fargate-logs-with-kinesis-data-firehose.md

```diff
@@ -0,0 +1,195 @@
+{{< /code-block >}}
+
+3. Use `kubectl` to apply the ConfigMap manifest.
+
+{{< code-block lang="bash" filename="" disable_copy="false" collapsible="false" >}}
+kubectl apply -f aws-logging-configmap.yaml
+{{< /code-block >}}
+
+4. Create an IAM policy and attach it to the pod execution role to allow the log router running on AWS Fargate to write to the Kinesis Data Firehose. You can use the example below, replacing the ARN in the **Resource** field with the ARN of your delivery stream.
+
+{{< code-block lang="json" filename="allow_kinesis_put_permission.json" disable_copy="false" collapsible="false" >}}
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "firehose:PutRecord",
+                "firehose:PutRecordBatch"
+            ],
+            "Resource":
+       "arn:aws:firehose:<REGION>:<ACCOUNTID>:deliverystream/<DELIVERY-STREAM-NAME>"
+       }
+]
+}
+{{< /code-block >}}
+
+   a. Create the policy.
+
+{{< code-block lang="bash" filename="" disable_copy="false" collapsible="false" >}}
+aws iam create-policy \
+         --policy-name FluentBitEKSFargate \
+         --policy-document file://allow_kinesis_put_permission.json 
+{{< /code-block >}}
+
+   b. Retrieve the Fargate Pod Execution Role and attach the IAM policy.
+
+{{< code-block lang="bash" filename="" disable_copy="false" collapsible="false" >}}
+ POD_EXEC_ROLE=$(aws eks describe-fargate-profile \
+   --cluster-name fargate-cluster \
+   --fargate-profile-name fargate-profile \
+   --query 'fargateProfile.podExecutionRoleArn' --output text |cut -d '/' -f 2)
+ aws iam attach-role-policy \
+         --policy-arn arn:aws:iam::<ACCOUNTID>:policy/FluentBitEKSFargate \
+         --role-name $POD_EXEC_ROLE
+{{< /code-block >}}
+### Deploy a sample application
+
+To generate logs and test the Kinesis pipeline, deploy a sample workload to your EKS Fargate cluster.
+
+1. Create a deployment manifest `sample-deployment.yaml`.
+ 
+{{< code-block lang="yaml" filename="sample-deployment.yaml" disable_copy="false" collapsible="false" >}}
+ apiVersion: apps/v1
+ kind: Deployment
+ metadata:
+   name: sample-app
+   namespace: fargate-namespace
+ spec:
+   selector:
+     matchLabels:
+       app: nginx
+   replicas: 1
+   template:
+     metadata:
+       labels:
+         app: nginx
+     spec:
+       containers:
+       - name: nginx
+         image: nginx
+         ports:
+         - containerPort: 80
+{{< /code-block >}}
+ 
+ 2. Create the `fargate-namespace` namespace.
+
+ {{< code-block lang="bash" filename="" disable_copy="false" collapsible="false" >}}
+ kubectl create namespace fargate-namespace
+ {{< /code-block >}}
+
+ 3. Use `kubectl` to apply the deployment manifest.
+
+ {{< code-block lang="bash" filename="" disable_copy="false" collapsible="false" >}}
+ kubectl apply -f sample-deployment.yaml
+ {{< /code-block >}}
+
```

Comment:
Would it be possible to add a short  `### Validation` section here letting users know how to confirm this is working?

Reply:
👍 Added some validation steps.  Please let me know if there's a better way to do the expected output blocks.

Reply:
Also, it's rebased on a recent master.